templates/app/docker-compose.yaml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: <%= appName %>
+    ports:
+      - "<%= port %>:<%= port %>"
```

Comment:
To be more standard and avoid the user to know how forest handle environment variable, I would not copy the .env file and configure the variable directly in the docker-compose.yaml.

Reply:
Me too!